### ISSUE TITLE
🐞 [Website UI] Nav menu links not visible on small media devices

### ISSUE THREAD

**Issue by gyancodes** [Atri-Labs/atrilabs-engine#718]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

 nav menu on mobile devices has a background transparent which blends with the background
![navissue](https://user-images.githubusercontent.com/61907167/223955512-3b105ac2-860f-4283-81f7-3c87d70920ac.PNG)

### Expected Behavior

![fixnavissue](https://user-images.githubusercontent.com/61907167/223955647-44ff6549-9919-4614-9063-0f0f10656803.PNG)

**Comment by haszankauna** [comment]
hello @gyancodes can I please be assigned this issue?

**Comment by darshitac11** [comment]
@haszankauna - I assigned it to you. Let me know if you need any help.

**Comment by gyancodes** [comment]
@darshitac11 Which is the main repo for the website?

**Comment by darshitac11** [comment]
Here it is: https://github.com/Atri-Apps/atri-landing

**Comment by NiazMorshed2007** [comment]
Hi there!
Is this still open?
I'd like to contribute.

**Comment by darshitac11** [comment]
Hi @NiazMorshed2007 - it's still open. Assigned you the issue. 

Let us know if you have any questions.

**Comment by NiazMorshed2007** [comment]
> Hi @NiazMorshed2007 - it's still open. Assigned you the issue.
> 
> Let us know if you have any questions.

Thanks I'm working on it,, but looks like it's the different repo, right?
https://github.com/Atri-Apps/atri-landing

**Comment by darshitac11** [comment]
Yes, it's a different repo - this repo has the code for framework only.

**Comment by Epheoluwa** [comment]
Any update on this bug?
Has it been resolved?
